app/services/dangling_account_remover.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Service for removing dangling accounts
+class DanglingAccountRemover
+  class << self
+    def run!
+      User.delete_all
```

Comment:
This should delete only the relevant User records. If the User table is discontinued, it should be removed first.

Reply:
We agreed with @akofink that we should delete all of these as we no longer use the records...

Reply:
True, I don't see why we shouldn't just remove the table altogether.

Reply:
On second thought, removing the table (and associated model) would cause issues with the way we store User.current.

Reply:
The model can stay, we can make the fields accessible by `attr_accessor` and get rid of the table :thinking: but maybe in a separate PR/issue.

Reply:
Some thoughts about users, not for this PR:

- Account.current in favor of `User.current = User.new(account: a)` seems to fit better with how we're using users/accounts
- Pundit, used for authorization, requires a user (not an account)
- We might need users again when we implement fine grained RBAC, but maybe not